packages/wdio-appium-service/tests/launcher.test.js

```diff
@@ -0,0 +1,155 @@
+import { AppiumLauncher } from '../src/launcher'
+import childProcess from 'child_process'
+import fs from 'fs-extra'
+import path from 'path'
+
+jest.mock('child_process')
```

Comment:
You can place the contents from the `__mocks__` directory here instead. An example can be found in wdio-selenium-standalone-service/tests/launcher.test.js. This way it will only mock child_process and fs for that file.

Reply:
I did that in the first place but moved it to the `__mocks__` directory after @christian-bromann's comment (https://github.com/webdriverio/webdriverio/pull/3717#discussion_r266889489).
I've moved them back to the test file.